### ISSUE TITLE
sharedClone with nn.BatchNormalization? 

### ISSUE THREAD

**Issue by dineshj1** [Element-Research/dpnn#44]
Hi, it looks like when I apply sharedClone to a nn.BatchNormalization layer, the running mean and running variance stored in this module are not shared across the clones. 

I can see some rationale for this, but not completely sure that this is intentional behavior.

**Comment by nicholas-leonard** [comment]
@dineshj1 This should be fixed in https://github.com/Element-Research/dpnn/commit/607bbcaf8b5133d1d5ec4d7883c62046467dc0b6 . Let me know if it works or not. Thanks for bringing this up.
